service/history/workflow/update/update.go

```diff
@@ -324,12 +326,15 @@ func (u *Update) Admit(
 			return
 		}
 		u.setState(stateAdmitted)
+		u.admittedTime = time.Now()
```

Comment:
It would be nice to pass time explicitly to `Admit` and get it from `TimeSource`. Generally, I would avoid using `time.Now()` anywhere in the code.

Reply:
I think a sequence ID is the most elegant solution but decided to use a timestamp for convenience for now. We're essentially using it as a source of increasing integers to make relative comparisons.

We chatted offline -- switched to `Now().UTC()` for now since the code base makes `TimeSource` inconvenient to obtain here.